cmd/cvp-trigger/README.md

```diff
@@ -70,6 +70,8 @@ CVP Trigger must load the configuration for the
 - `OO_CHANNEL`: Required. The name of the operator channel to track.
 - `OO_INSTALL_NAMESPACE`: Optional. The namespace into which the operator and
    catalog will be installed. If empty, a new namespace will be created.
+- `OO_PYXIS_URL`: Optional. Represents cvp product package name for specific ISV
```

Comment:
Is the description accurate? The variable name has `URL` in it, but the description says it represents a package name

Reply:
It is not currently. I need to update it. I thought we will be providing only the ISV_PID here, but we will provide whole URL together with the PID. Thanks for that catch!